### PR TITLE
Harden featureService

### PR DESCRIPTION
**Pull request by c-w**
See https://github.com/CatalystCode/project-fortis-pipeline/issues/239:

- Ensure packages are up to date
- Remote outdated deploy script
- Disable ssh password authentication
- Enable fail2ban

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -7,6 +7,7 @@ DUMP_VERSION='v2'
 # setup
 build_dependencies='curl git build-essential'
 sudo apt-get update > /dev/null
+sudo apt-get upgrade > /dev/null
```
Comment (anthturner) [inline scripts/install.sh]: Add `-y` to this line to avoid blocking on user input.

Comment (c-w) [reply scripts/install.sh]: Yep, got it locally, just need to push it.

Comment (c-w) [reply scripts/install.sh]: See 26fc2f1

**Comment by c-w** [comment]
Already tried this on a deployment and everything seems to be working fine :) http://fortis-features.eastus.cloudapp.azure.com/features/name/bogota

Thanks for the review!

**Pull request merged**